src/rgw/driver/sfs/sqlite/dbconn.h

```diff
@@ -117,7 +120,17 @@ inline auto _make_storage(const std::string &path) {
           sqlite_orm::make_column("id", &DBAccessKey::id, sqlite_orm::autoincrement(), sqlite_orm::primary_key()),
           sqlite_orm::make_column("access_key", &DBAccessKey::access_key),
           sqlite_orm::make_column("user_id", &DBAccessKey::user_id),
-          sqlite_orm::foreign_key(&DBAccessKey::user_id).references(&DBUser::user_id))
+          sqlite_orm::foreign_key(&DBAccessKey::user_id).references(&DBUser::user_id)),
+    sqlite_orm::make_table(std::string(LC_HEAD_TABLE),
+          sqlite_orm::make_column("lc_index", &DBOPLCHead::lc_index, sqlite_orm::primary_key()),
+          sqlite_orm::make_column("marker", &DBOPLCHead::marker),
+          sqlite_orm::make_column("start_date", &DBOPLCHead::start_date)),
+    sqlite_orm::make_table(std::string(LC_ENTRIES_TABLE),
+          sqlite_orm::make_column("lc_index", &DBOPLEntry::lc_index),
+          sqlite_orm::make_column("bucket_name", &DBOPLEntry::bucket_name),
+          sqlite_orm::make_column("start_time", &DBOPLEntry::start_time),
+          sqlite_orm::make_column("status", &DBOPLEntry::status),
+          sqlite_orm::primary_key(&DBOPLEntry::lc_index, &DBOPLEntry::bucket_name))
```

Comment:
is it possible for us to have the same `lc_index` value for multiple different `bucket_name` entries?

Reply:
In theory a bucket should only be found in a shard id. But I have to be honest... I'm not sure 100% of that affirmation and I added the composite key because the LC SAL layer is always querying the entry using the `lc_index` and the `bucket_name`.